docs/dxp/7.x/en/site-building/creating-pages/master-pages.md

```diff
@@ -1,27 +1,52 @@
 # Master Page Templates
 
-```note::
-  Master Page Templates are available since Liferay DXP 7.3.
-```
+> Liferay DXP 7.3+
+
+<!-- Feedback on this introductory session: It's resistant to skimming because of its length and how the information is structured/presented. Here's some suggestions, and you may have already guessed that one of those suggestions is...a table. One principle I consider as I reviewed this is that it feels like we're defining MPT's in the shadow of themes. When, I think if this article is primarily about MPT's - we should describe it standalone in its entirety and then provide the helpful contrast w/ Themes. If MPT's are "the future" then let's present them well, and only secondarily talk about how they relate to Themes.
```

Comment:
Good point. Since we're not pushing Themes as much in 7.3, I think we should only compare Master Page Templates and Themes as far as MPTs replace the Headers and Footers that were provided by Themes. So, I think we should leave out the Master Page Templates and Themes section you suggested and just focus on Master Page Templates for this.